lib/widget/form_helper.dart

```diff
@@ -13,17 +12,30 @@ import 'package:flutter/material.dart';
 class FormFieldController extends WidgetController {
   bool? enabled;
   bool required = false;
+  bool autoComplete = false;
   String? hintText;
   String? icon;
+  String? prefixIcon;
+  String? suffixIcon;
   String? iconLibrary;
   int? iconSize;
   int? iconColor;
   int? fontSize;
+  Color? borderColor;
+  Color? enabledColor;
+  Color? focusedColor;
+  Color? cursorColor;
+  double? boderRadius;
+  double? enabledBoderRadius;
+  TextStyle? hintStyle = EnsembleTheme.body2;
+  TextStyle? labelStyle = EnsembleTheme.body2;
+  double? focusedBoderRadius;
 
   @override
   Map<String, Function> getBaseGetters() {
     Map<String, Function> getters = super.getBaseGetters();
     getters.addAll({
+      'autoComplete': () => autoComplete,
```

Comment:
This is the wrong place. AutoComplete only make sense on dropdown. Why put it into the FormFieldController, which is the base controller for all form widgets?